app/controllers/admin/supplier_users_controller.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+class Admin::SupplierUsersController < Admin::BaseController
+  skip_after_action :verify_authorized, only: :index
+  skip_after_action :verify_policy_scoped, except: :index
+
+  before_action :set_suppliers, only: %i[new receive_supplier]
+  before_action :form_from_session, only: %i[user_details review create success]
+
+  def index
+    lead_providers = policy_scope(LeadProvider).joins(:users).includes(:users)
+    delivery_partners = policy_scope(DeliveryPartner).joins(:users).includes(:users)
```

Comment:
do we need both `join` and `includes` users?

Reply:
See https://github.com/DFE-Digital/early-careers-framework/pull/60#discussion_r564694120